src/Compiler/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorIntermediateNodeLoweringPhase.cs

```diff
@@ -236,15 +237,18 @@ private static void PostProcessImportedDirectives(DocumentIntermediateNode docum
 
     private struct UsingReference : IEquatable<UsingReference>
     {
-        public UsingReference(string @namespace, SourceSpan? source)
+        public UsingReference(string @namespace, SourceSpan? source, bool hasExplicitSemicolon = true)
```

Comment:
Is the default value needed? Consider removing it to make it clearer which call sites use which value.